user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserOperator.java

```diff
@@ -176,10 +177,13 @@ public KafkaUserOperator(Vertx vertx,
             scramOrNoneAcls = user.getSimpleAclRules();
         }
 
+        JsonObject newQuotasTls = user.isTlsUser() ? user.quotasToJson() : null;
```

Comment:
Stupid question, but why not just add it to lines 171-172 / 175-177 to have it next to where the same stuff is done for ACLs?

Reply:
Because I am blind :face_with_head_bandage: